sdcm/cluster_aws.py

```diff
@@ -279,6 +302,11 @@ def __init__(self, ec2_instance, ec2_service, credentials,
             self._ec2_service.create_tags(Resources=[self._instance.id],
                                           Tags=tags_list)
 
+    @property
+    def is_spot(self):
+        return True if (self._instance.instance_lifecycle and
```

Comment:
good practice of using the ternary conditional operator if when it is short and can fit in one line.
For better readability turn this in regular `if...else`